affine/__init__.py

```diff
@@ -465,39 +526,39 @@ def __gt__(self, other) -> bool:
     # the result is not an affine transform
 
     def __add__(self, other):
+        """Raise TypeError, since the add operation is not supported."""
```

Comment:
```suggestion
```

etc

Reply:
Ditto; this was added for [undocumented-magic-method (D105)](https://docs.astral.sh/ruff/rules/undocumented-magic-method/#undocumented-magic-method-d105)